packages/uui-card-content-node/lib/uui-card-content-node.element.ts

```diff
@@ -113,21 +113,37 @@ export class UUICardContentNodeElement extends UUICardElement {
     return html`<uui-icon .svg="${this.fallbackIcon}"></uui-icon>`;
   }
 
+  // This is deprecated - use href instead
+  #renderDeprecatedButton() {
+    return html`<div
+      id="open-part"
+      tabindex=${this.disabled ? (nothing as any) : 0}
+      @click=${this.handleOpenClick}
+      @keydown=${this.handleOpenKeydown}>
+      <span id="icon">
+        <slot name="icon" @slotchange=${this._onSlotIconChange}></slot>
+        ${this._iconSlotHasContent === false ? this._renderFallbackIcon() : ''}
+      </span>
+      <span id="name"> ${this.name} </span>
+    </div>`;
+  }
+
+  #renderLink() {
+    return html`<a
+      id="open-part"
+      tabindex=${this.disabled ? (nothing as any) : 0}
+      href=${this.href}>
```

Comment:
I would say we should stay aligned with UUI-button on the implementation.

So we are missing the ability to set 'target' and the part of auto setting 'rel' depending on that.